aws/devnet/variables.tf

```diff
@@ -0,0 +1,122 @@
+variable "farmer_reward_address" {
+  description = "Farmer's reward address"
+  type        = string
+}
+
+variable "domain_id" {
+  description = "Domain ID"
+  type        = list(number)
+  default     = [1, 2, 3]
+}
+
+variable "domain_labels" {
+  description = "Tag of the domain to run"
+  type        = list(string)
+  default     = ["evm", "payments"]
+}
+
+
+variable "bootstrap_node_ips" {
+  description = "IP of boostrap node"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "full_node_ips" {
+  description = "IP of boostrap node"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "rpc_node_ips" {
+  description = "IP of boostrap node"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "domain_node_ips" {
+  description = "IP of domain node"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "farmer_node_ips" {
+  description = "IP of boostrap node"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "instance_type" {
+  default = "t3.small"
+  type    = string
+}
+
+variable "vpc_id" {
+  default = "default"
+  type    = string
+}
+
+variable "azs" {
+  type        = list(string)
+  description = "Availability Zones"
+  default     = ["us-east-1a", "us-east-1b"]
+}
+
+variable "instance_count" {
+  type    = number
+  default = 1
+}
+
+variable "aws_region" {
+  description = "aws region"
+  type        = list(string)
+  default     = ["us-east-1"]
+}
+
+variable "public_subnet_cidrs" {
+  type        = list(string)
+  description = "Public Subnet CIDR values"
+  default     = ["172.31.1.0/24"]
+}
+
+variable "private_subnet_cidrs" {
+  type        = list(string)
+  description = "Private Subnet CIDR values"
+  default     = ["172.31.2.0/24"]
+}
+
+variable "disk_volume_size" {
+  type    = number
+  default = 50
```

Comment:
sure, will do that, I set these as a precaution of smaller values so mistakenly don't create large volumes and incur extra charges, the size of disks are set in the module configs.

Reply:
If they are set explicitly anyway, why having a default?

Reply:
I have removed the default. The variable will be set in tfvars file to desired amount needed.

Reply:
Well, this is exactly the problem I am trying to explain to you. Since this is a default value, it should not be required. If something else doesn't work after you remove default value, it seems to indicate you were in fact using default value, which as we already discussed is way too low and should be multiple times higher and specified explicitly.